content/post/know-your-aws-community-day-india-2020-speaker-yogesh-sharma.md

```diff
@@ -0,0 +1,27 @@
++++
+author = "Dhruv Aggarwal"
+date = 2020-09-23T19:02:16Z
+description = "Interested to know more about Yogesh Sharma? Read on."
+feature_image = "/images/yogesh.PNG"
+tags = ["AWS", "AWSCommunityDay2020", "Speaker", "Kinesis", "Fargate", "CFN"]
+title = "Know Your AWS Community Day India 2020 Speaker -Yogesh Sharma"
+
++++
+Yogesh Sharma is working as a Cloud Architect at TCS, Australia will be speaking on "**Intelligent Serverless and Scalable Real-Time Data Pipeline using Kinesis, Fargate, and CFN**" from 10:00 AM - 10:45 AM Track 2 on 16th October 2020 at AWS Community Day India 2020.
```

Comment:
>  working as [...] will be speaking on 


is not grammatically right. This should be 

`<name>` is currently `<title>` at `<company>`. `<First name>` will be speaking on <session title>....

No need to mention Australia

Reply:
Corrected!